src/cloud/components/organisms/settings/UserPreferencesForm.tsx

```diff
@@ -100,6 +108,41 @@ const UserPreferencesForm = () => {
     [setSettings]
   )
 
+  const updateFontSize: ChangeEventHandler<HTMLInputElement> = useCallback(
+    (event) => {
+      setFontSize(event.target.value)
+    },
+    [setFontSize]
+  )
+  useDebounce(
```

Comment:
- Changing font family loses focus after update (not sure why it happens here and not in local space)

@Rokt33r any ideas?

Reply:
Our usage of react select might cause the issue.